### PR TITLE
fix: style bugs

### PR DESCRIPTION
**Pull request by ding-js**

### CODE REVIEW

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

## 3.3.1 (https://github.com/Tradeshift/react-components/compare/v3.3.0...v3.3.1) (2019-09-06)

### Bug Fixes

 - style bugs (408ab77 (https://github.com/Tradeshift/react-components/commit/408ab77))

File: package.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
     "postcss-preset-env": "^6.7.0",
     "prettier": "^1.18.2",
     "prompts": "^2.2.1",
+    "prop-types": "^15.7.2",
```
Comment (LealLee) [inline package.json]: Not necessary here, it is already in `peerDependencies`

Comment (ding-js) [reply package.json]: > Not necessary here, it is already in `peerDependencies`

`peerDependencies ` is not using for develop

**Pull request merged**